lms/djangoapps/courseware/views/views.py

```diff
@@ -836,7 +836,7 @@ def _get_cert_data(student, course, course_key, is_active, enrollment_mode):
         return CertData(
             CertificateStatuses.unverified,
             'Certificate unavailable',
-            'You have not received a certificate because you do not have a current {platform_name} verified '
+            u'You have not received a certificate because you do not have a current {platform_name} verified '
```

Comment:
Since this doesn't look like a test code, should the two lines here (838 & 839) be enclosed by `_("")` to make it translatable?